cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -77,6 +80,27 @@ public Integer get() {
     }
   };
 
+  /*
+   * Wrapper class for reading/writing of username + accessToken
+   */
+  private class UserAccessToken {
```

Comment:
Now I understand what this code does. this class is more like a Client side info (like cookies in browser) to save states across multiple runs of the CLI. It's ok to keep this class as it (probably need to rename to something more appropriate when more fields are added in future).